packages/devtools_app/lib/src/screens/memory/panes/chart/controller/chart_data.dart

```diff
@@ -9,11 +9,15 @@ import '../../../../../shared/primitives/simple_items.dart';
 import '../../../shared/primitives/memory_timeline.dart';
 import '../data/primitives.dart';
 
-class _Json {
+// ignore: avoid_classes_with_only_static_members, enum-like classes are ok
+@visibleForTesting
+class Json {
   static const isDeviceAndroid = 'isAndroid';
   static const timeline = 'timeline';
   static const interval = 'interval';
   static const isLegendVisible = 'isLegendVisible';
+
+  static final all = [isDeviceAndroid, timeline, interval, isLegendVisible];
```

Comment:
make this a set for consistency above. Or preferably, just switch to using an enum so that you can use `Json.values`.

If you leave the `all` field, mark visible for testing if that is the only place it is used. Here and elsewhere.